crates/driver/src/domain/competition/mod.rs

```diff
@@ -238,7 +242,7 @@ impl Competition {
 /// ranking happens.
 #[derive(Debug)]
 pub struct Solved {
-    pub score: Score,
+    pub score: RankingScore,
```

Comment:
To me, the `competition::Score` is the "ranking score" - i.e. the score for the solver competition.

The `SolverScore` type makes sense (i.e. how the solver communicates how its score should be computed).

Reply:
Fixed https://github.com/cowprotocol/services/pull/1856/commits/7951cd10b758d0239d021ae4577e18070885d841